plugin/src/main/kotlin/tanvd/grazi/ide/fus/GraziFUCounterCollector.kt

```diff
@@ -0,0 +1,27 @@
+package tanvd.grazi.ide.fus
+
+import com.intellij.internal.statistic.eventLog.FeatureUsageData
+import com.intellij.internal.statistic.service.fus.collectors.FUCounterUsageLogger
+import tanvd.grazi.grammar.Typo
+import tanvd.grazi.language.Lang
+
+object GraziFUCounterCollector {
+    private fun log(eventId: String, body: FeatureUsageData.() -> Unit) = FUCounterUsageLogger.getInstance()
+            .logEvent("grazi", eventId, FeatureUsageData().apply(body))
```

Comment:
`FeatureUsageData().apply(body)` -> `FeatureUsageData().body()`

Reply:
To write this we need body to return a FeatureUsageData, so no changed.